modules/dispatcher/doc/dispatcher_admin.xml

```diff
@@ -1350,6 +1350,20 @@ onreply_route {
 </programlisting>
 		</example>
  	</section>
+	<section id="dispatcher.f.reload">
+		<title>
+		<function moreinfo="none">ds_reload()</function>
+		</title>
+		<para>
+		Reloads the groups and included destinations. The command is
+		disabled for call load based dispatching (algorithm 10) since
```

Comment:
This is based on [this](https://github.com/kamailio/kamailio/blob/master/modules/dispatcher/dispatcher.c#L914-L924) and [this](https://github.com/kamailio/kamailio/blob/master/modules/dispatcher/doc/dispatcher_admin.xml#L1415-L1437). I copied the possible problems description using a similar function.